lib/sycamore/sycamore/transforms/table_structure/table_transformers.py

```diff
@@ -1045,7 +1047,95 @@ def structure_to_cells(table_structure, tokens):
         cell_rect = row_rect.intersect(column_rect)
         if cell_rect.area > 0:
             cell["bbox"] = cell_rect.to_list()
-
+    if union_tokens:
```

Comment:
Can I get a block of comment describing the algorithm we're implementing?

Reply:
Done.

Reply:
Done.